packages/mdc-snackbar/README.md

```diff
@@ -108,69 +141,164 @@ This will move the snackbar offscreen until the CSS is fully loaded and avoids a
 CSS Class | Description
 --- | ---
 `mdc-snackbar` | Mandatory. Container for the snackbar elements.
-`mdc-snackbar__action-wrapper` | Mandatory. Wraps the action button.
-`mdc-snackbar__action-button` | Mandatory. The action button.
-`mdc-snackbar__text` | Mandtory. The next of the snackbar.
-`mdc-snackbar--align-start` | Optional. Class to align snackbar to start, ltr dependent.
-`mdc-snackbar--action-on-bottom` | Optional on the mdc-snackbar element. Moves action to bottom of snackbar. Can be applied in js.
-`mdc-snackbar--multiline` | Optional on the mdc-snackbar element. Makes the snackbar multiple lines. Can be applied in js.
+`mdc-snackbar__label` | Mandatory. Message text.
+`mdc-snackbar__actions` | Optional. Wraps the action button/icon elements, if present.
+`mdc-snackbar__action-button` | Optional. The action button.
+`mdc-snackbar__action-icon` | Optional. The dismiss ("X") icon.
+`mdc-snackbar--opening` | Optional. Indicates that the snackbar is open and visible.
```

Comment:
I think this description needs to swapped with `--open`.

Reply:
Done